installer/functions

```diff
@@ -239,3 +239,57 @@ websocketcommand() {
         echo "EError timeout"
     fi
 }
+
+# Check space requirements on CHROOTS partition
+# $1: the partition's minimum size threshold
+# $2: the partition's minimum available space threshold
+check_space() {
+    local part="${CHROOTS:-/usr/local}"
+    df -m $part | awk 'FNR == 2 {print $2,$4}' | {
+        read -r size avail
+            if [ "$size" -lt "$1" ]; then
+                local size_gb="$((${size}/1000))GB"
+                echo "WARNING: Your CHROOTS partition is rather small (${size_gb})."
```

Comment:
I think we've been outputting warnings like these out to stderr, but might want to check for consistency